<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-R2K6MVF" failures="8" tests="17" name="Command line test" time="0.121" errors="0" timestamp="2021-04-27T09:00:32 ICT">
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testGetHinh" time="0.031">
    <failure type="java.lang.AssertionError" message="expected [] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [] but found [null]
at edu.poly.model.ChuyenDeNGTest.testGetHinh(ChuyenDeNGTest.java:169)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testGetHinh -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testGetHocPhi" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testGetMaCD" time="0.016">
    <failure type="java.lang.AssertionError" message="expected [] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [] but found [null]
at edu.poly.model.ChuyenDeNGTest.testGetMaCD(ChuyenDeNGTest.java:46)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testGetMaCD -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testGetMoTa" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [] but found [null]
at edu.poly.model.ChuyenDeNGTest.testGetMoTa(ChuyenDeNGTest.java:190)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testGetMoTa -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testGetTenCD" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [] but found [null]
at edu.poly.model.ChuyenDeNGTest.testGetTenCD(ChuyenDeNGTest.java:66)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testGetTenCD -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testGetThoiLuong" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetHinh" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetHocPhi" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetHocPhiWithNegative" time="0.0">
    <failure type="org.testng.TestException" message="
Method ChuyenDeNGTest.testSetHocPhiWithNegative()[pri:0, instance:edu.poly.model.ChuyenDeNGTest@72d1ad2e] should have thrown an exception of type class java.lang.Exception">
      <![CDATA[org.testng.TestException: 
Method ChuyenDeNGTest.testSetHocPhiWithNegative()[pri:0, instance:edu.poly.model.ChuyenDeNGTest@72d1ad2e] should have thrown an exception of type class java.lang.Exception
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testSetHocPhiWithNegative -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetHocPhiWithZero" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetMaCD" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetMoTa" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetTenCD" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetThoiLuong" time="0.0"/>
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetThoiLuongWithNegative" time="0.0">
    <failure type="org.testng.TestException" message="
Method ChuyenDeNGTest.testSetThoiLuongWithNegative()[pri:0, instance:edu.poly.model.ChuyenDeNGTest@72d1ad2e] should have thrown an exception of type class java.lang.Exception">
      <![CDATA[org.testng.TestException: 
Method ChuyenDeNGTest.testSetThoiLuongWithNegative()[pri:0, instance:edu.poly.model.ChuyenDeNGTest@72d1ad2e] should have thrown an exception of type class java.lang.Exception
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testSetThoiLuongWithNegative -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testSetThoiLuongWithZero" time="0.0">
    <failure type="org.testng.TestException" message="
Method ChuyenDeNGTest.testSetThoiLuongWithZero()[pri:0, instance:edu.poly.model.ChuyenDeNGTest@72d1ad2e] should have thrown an exception of type class java.lang.Exception">
      <![CDATA[org.testng.TestException: 
Method ChuyenDeNGTest.testSetThoiLuongWithZero()[pri:0, instance:edu.poly.model.ChuyenDeNGTest@72d1ad2e] should have thrown an exception of type class java.lang.Exception
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- testSetThoiLuongWithZero -->
  <testcase classname="edu.poly.model.ChuyenDeNGTest" name="testToString" time="0.0">
    <failure type="java.lang.AssertionError" message="expected [] but found [null]">
      <![CDATA[java.lang.AssertionError: expected [] but found [null]
at edu.poly.model.ChuyenDeNGTest.testToString(ChuyenDeNGTest.java:211)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1540)
... Removed 42 stack frames]]>
    </failure>
  </testcase> <!-- testToString -->
</testsuite> <!-- Command line test -->
